src/config.tsx

```diff
@@ -0,0 +1,15 @@
+const defaultConfig: Config = {
+  globalTree: true,
+  panels: [
+    {
+      entrypoint:
```

Comment:
Hm I thought you would implement this in some next PR. This was just an idea, I'm not sure about this yet.